src/scripts/components/layer/wv.layer.list.js

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import LayerRadio from './wv.layer.radio.js';
+import { CellMeasurer, CellMeasurerCache, List } from 'react-virtualized';
+
+/*
+ * A react component, Builds a list of layers using the LayerRadio component
+ * @class LayerList
+ * @extends React.Component
+ */
+export default class LayerList extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      layerFilter: props.config.layerOrder,
+      expandedLayers: []
+    };
+    this._cache = new CellMeasurerCache({
+      fixedWidth: true,
+      minHeight: 23,
+      minWidth: props.width
+    });
+  }
+
+  /*
+   * Clear cell catch if component will
+   *  rerender
+   * @method componentWillUpdate
+   * @return {void}
+   */
+  componentWillUpdate(){
+    this._cache.clearAll();
+  }
+
+  /*
+   * Saves the visibility/state of the metadata for each layer
+   * @method saveExpandedInfoState
+   * @param {string} layerID - the layer which has metadata that
```

Comment:
Proper camel casing of this would be `layerId`, and the param is called `layer` in the function, so we should rename it in the function to `layerId` to be clearer and consistent with the comment.